shared/bee-game.js

```diff
@@ -112,10 +112,68 @@ function updateBoids(boids, { players }) {
   const teams = teamIds.map(teamId => ({
     boids: boidsAndPlayers.filter(boid => boid.teamId === teamId),
     allies: players.filter(player => player.teamId === teamId),
-  }))
+  }));
+
+  const { winner, loser } = duel(teams);
+  takeOverNearestBoid(winner, loser);
+
   return boids.map((boid) => updateBoid(boid, teams[boid.teamId]));
 }
 
+function duel(teams) {
+  if ( teamSpread(teams[0]) > teamSpread(teams[1]) ) {
+    return {
+      winner: teams[1],
+      loser: teams[0]
+    };
+  } else {
+    return {
+      winner: teams[0],
+      loser: teams[1]
+    };
+  }
+}
+
+function teamSpread(team) {
+  const xCoords = team.boids.map(b => b.position.x);
+  const yCoords = team.boids.map(b => b.position.y);
+  const boundingBox = {
+    width: Math.max(xCoords) - Math.min(xCoords),
```

Comment:
I'm not sure `Math.max(array)` is possible. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/max says `Math.max(...array)` is needed.